frontend/src/metabase/actions/components/ActionViz/Action.unit.spec.tsx

```diff
@@ -254,6 +290,71 @@ describe("Actions > ActionViz > Action", () => {
         });
       });
     });
+
+    it("should allow to edit underlying action if a user has edit permissions", async () => {
```

Comment:
We also need to test that you can't edit an action if you don't have permissions